mon-pix/app/routes/campaigns/assessment/tutorial.js

```diff
@@ -3,16 +3,16 @@
 
 import Route from '@ember/routing/route';
 import SecuredRouteMixin from 'mon-pix/mixins/secured-route-mixin';
-import campaignTutorial from 'mon-pix/static-data/campaign-tutorial';
 import { inject as service } from '@ember/service';
 
 export default Route.extend(SecuredRouteMixin, {
 
   currentUser: service(),
+  intl: service(),
 
   campaignCode: null,
   tutorialPageId: 0,
-  tutorial: campaignTutorial.tutorial,
+  tutorialPageCount: 4,
```

Comment:
Je suis perplexe sur l'assignation de cette variable à une constante.
Que se passe-t-il si demain le nombre de pages du didacticiel change ?
Il ne faudra surtout pas oublier de modifier la valeur.
Ne peut-on pas se baser sur la clé de traduction ?

Reply:
Une idée pour la clé de trad ? 
Nous n'avons pas encore trouvé de moyen facile de récupérer cette info via `this.intl`
On s'est aussi dit qu'on pourrait mettre cette valeur dans une clé des fichiers `fr.json` et `en.json`, mais cette utilisation n'est pas celle de la traduction, et ça serait une utilisation déviante de ce fichier.

Reply:
On doit pouvoir lire le fichier json directement.
```
const translations = require('fr.json');

const tutorialPageCount = Object.keys(translations.pages.tutorial.pages).length;
```